### ISSUE TITLE
Snackbar index.d.ts error

### ISSUE THREAD

**Issue by dave-hillier** [jamesmfriedman/rmwc#345]
I'm unable to use Snackbar from typescript in 3.0.5. I get the following error. 

```
(6,29): Could not find a declaration file for module '@material/snackbar/dist/mdc.snackbar'. '/Users/daveh/source/repos/housekeeper-service/service/ClientApp/node_modules/@material/snackbar/dist/mdc.snackbar.js' implicitly has an 'any' type.
        Try `npm install @types/material__snackbar` if it exists or add a new declaration (.d.ts) file containing `declare module '@material/snackbar/dist/mdc.snackbar';`
```
As a workaround I removed line 6 @rmwc/snackbar/index.d.ts and replaced MDCSnackbar on line 120 with object.

**Comment by dave-hillier** [comment]
I don't understand how the types are generated otherwise I'd try help

**Comment by jamesmfriedman** [comment]
The problem is actually upstream in definitely typed. They are apparently lacking a definition for the snack bar module.

**Comment by jamesmfriedman** [comment]
You can leave this open. It needs to be fixed!

**Comment by dave-hillier** [comment]
One difference to note is the `syncWithProps` method. In these examples it uses PropsT 

https://github.com/jamesmfriedman/rmwc/blob/master/src/slider/index.js#L130
https://github.com/jamesmfriedman/rmwc/blob/master/src/tabs/index.js#L78
That isL `syncWithProps(nextProps: TabBarPropsT) {`

However, in snackbar it's different:
 https://github.com/jamesmfriedman/rmwc/blob/master/src/snackbar/index.js#L118
`syncWithProps(nextProps: MDCSnackbar) {`
Should it be `SnackbarPropsT`?

**Comment by jamesmfriedman** [comment]
Fixed with 3.0.7
